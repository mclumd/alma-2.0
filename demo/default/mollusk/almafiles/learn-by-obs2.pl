obs(quote(not(rel(shell_bearer, bob)))).
obs(quote(rel(cephalopod, bob))).
