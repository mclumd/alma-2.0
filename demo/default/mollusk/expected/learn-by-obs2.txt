~rel(shell_bearer, bob)
rel(cephalopod, bob)
rel(mollusk, bob)
abnormal(bob, mollusk, "~rel(shell_bearer, bob)")
